.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,101 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - release
+      - hotfix
+  pull_request:
```

Comment:
[ASK] should we start activating the merge queues ?

Reply:
If we do so, this might be done in a dedicated PR